crates/zed/src/languages/nickel/injections.scm

```diff
@@ -0,0 +1,3 @@
+(annot_atom doc: (static_string)
+            @injection.content
+            (#set! injection.language "markdown"))
```

Comment:
I don't think injections are working. Let me play around with this.

Reply:
I think I fixed it. Adding a new line inside a doc coment after a `#` no longer inserts another `#` in the new line (it's markdown, not a code comment).